src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private Node[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        size = 0;
+    }
+
+    private class Node<K, V> {
+        private int hash;
```

Comment:
```suggestion
        private int hash;
```
is null everywhere

Reply:
I keep hashCode in variable hash because this variable exists in original HashMap but don't use it.